x-pack/test/alerting_api_integration/spaces_only/tests/alerting/alerts.ts

```diff
@@ -32,38 +37,41 @@ export default function alertTests({ getService }: FtrProviderContext) {
       await es.indices.delete({ index: authorizationIndex });
     });
 
-    async function waitForTestIndexDoc(source: string, reference: string) {
-      return await retry.try(async () => {
-        const searchResult = await es.search({
-          index: esTestIndexName,
-          body: {
-            query: {
-              bool: {
-                must: [
-                  {
-                    term: {
-                      source,
-                    },
+    async function searchTestIndexDocs(source: string, reference: string) {
```

Comment:
nit: potentially another candidate method for a rollup object for the testIndex "tool" mentioned previously - `search()`